pkg/debug/start_debug.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package debug
+
+import (
+	ctx "context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	appsv1 "k8s.io/api/apps/v1"
+	autoscalingv1 "k8s.io/api/autoscaling/v1"
+	corev1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func StartDebug(context *k8sutil.Context, clusterNamespace, deploymentName, alternateImageValue string) {
```

Comment:
It would be nice to exit in a single place, for example with this simple wrapper method. Then the wrapped method just returns the errors. Perhaps even better if the wrapper method were in the `cmd` package, and methods under `pkg` return the errors.
```suggestion
func StartDebug(context *k8sutil.Context, clusterNamespace, deploymentName, alternateImageValue string) {
{
  err := startDebug()
  if err != nil {
    // log the error
    // exit
  }
}
func startDebug(context *k8sutil.Context, clusterNamespace, deploymentName, alternateImageValue string) error 
```

Reply:
done